packages/ui/src/machines/authenticator/actors/signIn.ts

```diff
@@ -841,7 +765,7 @@ export function signInActor({ services }: SignInMachineOptions) {
           );
         },
         async getCurrentUserResolved(context) {
-          groupLog('+++getCurrentUserResolved', context, event);
+          groupLog('+++getCurrentUserResolved', context);
```

Comment:
```suggestion
```

Reply:
Let's remove these logs after we're stable.

Reply:
Same could be said for removing the commented code that was left to give us a better understanding of the code we are adding?

Reply:
Only removed commented out code that already had implementation in place and I have tested. Additionally we have a ref of that on main if need be.

Reply:
Just looking for consistency, please choose one or the other. Do not care which